unity-renderer/Assets/DCLServices/MapRendererV2/MapLayers/PlayerMarker/PlayerMarkerObject.cs

```diff
@@ -1,10 +1,23 @@
-﻿using UnityEngine;
+﻿using DCL;
+using UnityEngine;
 
 namespace DCLServices.MapRendererV2.MapLayers.PlayerMarker
 {
     internal class PlayerMarkerObject : MapRendererMarkerBase
     {
-        [field: SerializeField]
-        internal SpriteRenderer spriteRenderer { get; private set; }
+        [SerializeField] private SpriteRenderer compass;
+        [SerializeField] private SpriteRenderer circle;
+
+        private void Start()
+        {
+            if (!DataStore.i.featureFlags.flags.Get().IsFeatureEnabled("map_focus_home_or_user"))
```

Comment:
this is a bit ugly, maybe we could enavke/disable the circle from the player marker controller throught the `playermarker.cs`